### PR TITLE
Add onboarding blog entry for outcomerate

### PR DESCRIPTION
**Pull request by rtaph**
Dear @stefaniebutland, please have a look and let me know if you have thoughts. I left the topicid blank in the YAML.

Thanks,

### CODE REVIEW

**Comment by stefaniebutland** [comment]
Thanks @rtaph I'll have a look and get you feedback Friday!

File: content/blog/2018-10-02-outcomerate.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+The package also implements weighted calculations of rates. Weighted figures further advance reproducible research because they (usually) estimate the outcome rate at the population level rather than for the sample at hand. This is useful information to share with the world, as it helps others plan for sample losses in future studies. Unless you plan to replicate the sampling design exactly, it is often more useful to know the (weighted) population rate.
+
+## Future Development
+
+In future releases of [`outcomerate`](https://github.com/ropensci/outcomerate), I may implement other sections from the AAPOR manual. Though arguably less popular than the “main” outcome rates, AAPOR has separate chapters for telephone surveys, online panels, and mixed-mode survey work. I chose not implement these initially, because some of the definitions differ despite using similar codes. It may also be worthwhile to add functions that make professional, publication-ready tables of the outcome rates. I am also considering functions that provide the standard error of the rates.
```
Comment (stefaniebutland) [inline content/blog/2018-10-02-outcomerate.md]: Could you turn a couple of these into issues that others might consider working on?  Some may be code, but maybe others could be vignettes.
If yes, could you label for Hacktoberfest ? (See https://twitter.com/rOpenSci/status/1044708122265350145, https://hacktoberfest.digitalocean.com/)

File: content/blog/2018-10-02-outcomerate.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Future development may also see more vignettes. Some topics I wish to write more about include real-time monitoring of survey quality (using Empirical Bayes methods), and variance calculations of outcome rates in complex designs.
+
+## Acknowledgements
+
+My thanks goes to [Neil Richardson](https://github.com/nealrichardson), [Carl Ganz](https://github.com/carlganz), and [Karthik Ram](https://github.com/karthik) for their peer review and editorial support at [ROpenSci](https://ropensci.org/). [`outcomerate`](https://github.com/ropensci/outcomerate) is better for it my programming style and knowledge has improved. Naturally, I also wish to acknowledge AAPOR for advancing the practice of survey research— this package is built on the definitions they helped standardize.
```
[the remark this answers is not in the document]
Comment (stefaniebutland) [reply content/blog/2018-10-02-outcomerate.md]: add punctuation or conjunction after "better for it..."

[the remark this answers is not in the document]
Comment (stefaniebutland) [reply content/blog/2018-10-02-outcomerate.md]: thank you for acknowledging the reviewers and editor. We'll tag reviewers in a thank you tweet.

File: content/blog/2018-10-02-outcomerate.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+The philosophy of the package is simple: let users flexibly calculate as many rates as possible with minimal effort, and fail fast. Unless specifically requested, the function only returns the rates that are defined based on the input, and provides soft warnings when it suspects users may be making errors. My hope is that this will encourage the frequent use of standard definitions, and facilitate their correct application.
+
+The package also implements weighted calculations of rates. Weighted figures further advance reproducible research because they (usually) estimate the outcome rate at the population level rather than for the sample at hand. This is useful information to share with the world, as it helps others plan for sample losses in future studies. Unless you plan to replicate the sampling design exactly, it is often more useful to know the (weighted) population rate.
+
```
Comment (stefaniebutland) [inline content/blog/2018-10-02-outcomerate.md]: I think readers would be keen for a taste of outcomerate's usage. Could you link to the example https://github.com/ropensci/outcomerate#example?

File: content/blog/2018-10-02-outcomerate.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
+slug: "outcomerate"
+title: Honest Assessments of Survey Quality with AAPOR Outcome Rates
```
Comment (stefaniebutland) [inline content/blog/2018-10-02-outcomerate.md]: Is there a way you could modify the title to reveal more of what `outcomerate` does? i.e. I didn't know what AAPOR was but seeing the "public opinion research" part of American Association of Public Opinion Research intrigued me. Without that, I didn't have a sense that this was social sciences related (we have a growing community of people working in R for social sciences) or what type of outcome rates it deals with (ecological field research maybe ;-) ?)

I also note that the definition of AAPOR does not come till second section "What Problem Does the Package Solve?" which makes it more impt to try to get the info out earlier, and in title if possible

File: content/blog/2018-10-02-outcomerate.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+- package
+- outcomerate
+- reproducible-research
+- survey
+output: html_document
+---
+
+## Background
+
+Surveys are ubiquitous in the social sciences, and the best of them are meticulously planned out. Statisticians often decide on a sample size based on a theoretical design, and then proceed to inflate this number to account for "sample losses". This ensures that the desired sample size is achieved, even in the presence of non-response. Factors that reduce the pool of interviews include participant refusals, inability to contact respondents, deaths, and frame inaccuracies. The more non-response, the more a study becomes open to criticism about its veracity.
```
Comment (stefaniebutland) [inline content/blog/2018-10-02-outcomerate.md]: can you clarify what "frame inaccuracies" are for an outside reader?

File: content/blog/2018-10-02-outcomerate.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+![](/img/blog-images/2018-10-02-outcomerate/samples.jpg)
+
+To simplify communication about non-response, a set of "outcome rates" have emerged over time. These rates are essentially quality declarations about the operational elements a survey. Broadly speaking, they relate how well an intended sample _s_ matches the actual respondent set _r_.
```
Comment (stefaniebutland) [inline content/blog/2018-10-02-outcomerate.md]: "elements a survey"  elements *in* a survey?

File: content/blog/2018-10-02-outcomerate.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+Though no such R package existed previously, AAPOR does make available a [excel-based calculator](https://www.aapor.org/Education-Resources/For-Researchers/Poll-Survey-FAQ/Response-Rates-An-Overview.aspx). It is a wonderful free tool, but has some limitations. For one, the excel calculator is not designed to compute many rates by sub-domain; In my own work, I often try need rates in real time by day, by researcher/enumerator, and by geography. Doing each analysis in a separately excel workbook is impractical. Instead, [`outcomerate`](https://github.com/ropensci/outcomerate) makes it easy to achieve this with `dplyr`, `aggregate()`, or `data.table`. Second, Excel is  notorious for failing silently. It will give you a result even if the input contains errors or inconsistencies. This is dangerous because mistakes can creep into your work without your knowledge.
```
Comment (stefaniebutland) [inline content/blog/2018-10-02-outcomerate.md]: in "sub-domain; In my own work" change punctuation or do not capitalize "In"

File: content/blog/2018-10-02-outcomerate.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+- reproducible-research
+- survey
+output: html_document
+---
+
+## Background
+
+Surveys are ubiquitous in the social sciences, and the best of them are meticulously planned out. Statisticians often decide on a sample size based on a theoretical design, and then proceed to inflate this number to account for "sample losses". This ensures that the desired sample size is achieved, even in the presence of non-response. Factors that reduce the pool of interviews include participant refusals, inability to contact respondents, deaths, and frame inaccuracies. The more non-response, the more a study becomes open to criticism about its veracity.
+
+![](/img/blog-images/2018-10-02-outcomerate/samples.jpg)
```
Comment (stefaniebutland) [inline content/blog/2018-10-02-outcomerate.md]: this fig is very helpful

File: content/blog/2018-10-02-outcomerate.md (hunk 1 above)
Comment (rtaph) [reply content/blog/2018-10-02-outcomerate.md]: Excellent suggestion. I will prepare some issues and link them into the blog with an invitation to contribute.

File: content/blog/2018-10-02-outcomerate.md (hunk 5 above)
Comment (rtaph) [reply content/blog/2018-10-02-outcomerate.md]: Yes, good point. I will add footnote.

File: content/blog/2018-10-02-outcomerate.md (hunk 4 above)
Comment (rtaph) [reply content/blog/2018-10-02-outcomerate.md]: Point very well taken. What do you think about _"outcomerate: Transparent Communication of Quality in Social Surveys"_ instead?

File: content/blog/2018-10-02-outcomerate.md (hunk 3 above)
Comment (rtaph) [reply content/blog/2018-10-02-outcomerate.md]: Yes, great point. In reviewing it now I am actually working in simple examples directly into the text.

**Comment by rtaph** [comment]
@stefaniebutland I implemented all your suggestions. The only things left are the YAML topicid and, optionally, linking some examples from the vignettes if I manage to get the pkgdown site running.

**Comment by rtaph** [comment]
@stefaniebutland the latest recommendations sound good to me. Let's wait and see if we get a hold of Karthik before the evening of Oct 1. Linking the examples would be nice but not critical, so I updated the PR with the other fixes. Is it your preference for me to squash my commits before we finalize?

My twitter handle is [@rafaeltaph](https://twitter.com/rafaeltaph)

**Comment by stefaniebutland** [comment]
> Is it your preference for me to squash my commits before we finalize

I will squash and merge when I publish.
Karthik says he's done his thing

**Comment by rtaph** [comment]
All done and ready to go, @stefaniebutland. The blog entry now points to examples in the vignettes.

On a closing note, I have really enjoyed this process. Thanks for all the constructive, positive support :)

**Pull request merged**